.taskcluster.yml

```diff
@@ -0,0 +1,31 @@
+version: 0
+metadata:
+  name: servo
+  description: >-
+    A modern, high-performance browser engine designed for both application
+    and embedded use.
+  owner: '{{ event.head.user.email }}'
+  source: '{{ event.head.repo.url }}'
+tasks:
+  - provisionerId: '{{ taskcluster.docker.provisionerId }}'
+    workerType: '{{ taskcluster.docker.workerType }}'
+    extra:
+      github:
+        events: []
+    payload:
+      maxRunTime: 3600
+      image: servobrowser/servo-linux-dev
```

Comment:
I'd prefer to not pin this quite yet, as we may need to iterate on the image due to TaskCluster failures and using the latest will save us a lot of spurious commits to the Servo repo. However, we should pin it before we consider gating on TaskCluster results.